### DISCUSSION TITLE
Mudar as colunas das tabelas

### DISCUSSION THREAD

**Discussion by Steven-Esteves** [RamonSilva20/mapos#1812] [Q&A] [answered]
Boas, estou a tentar trabalhar com a aplicação para uma empresa e precisava de alterar ou criar novas colunas na tabela dos Clientes, mas se apagar as colunas lá inseridas dá me error, e se fizer colunas novas não consigo que os dados fiquem lá guardados. 
Algem me sabe explicar como fazer isso?
Obrigado.

**Comment by Steven-Esteves** [comment, marked as answer]
Já resolvi, Muito Obrigado na mesma!

**Comment by Pr3d4dor** [comment]
@Steven-Esteves Qual era o problema, se puder responder, pode ajudar alguém com o mesmo problema
